app/views/public_activity/webhook/_create.html.slim

```diff
@@ -0,0 +1,23 @@
+li
+  .activitie-container
```

Comment:
If so, it seems to be wrong in other places as well, e.g. `app/views/public_activity/namespace/_create.html.slim`. Not sure if this matters though.